pkg/metadata/inventories/config.go

```diff
@@ -0,0 +1,15 @@
+package inventories
+
+import "github.com/DataDog/datadog-agent/pkg/config"
+
+// SetConfigMetadata sets the agent metadata based on the given configuration
+func SetConfigMetadata(cfg config.Config) {
+	SetAgentMetadata(AgentConfigApmDDURL, cfg.GetString("apm_config.dd_url"))
+	SetAgentMetadata(AgentConfigDDURL, cfg.GetString("dd_url"))
+	SetAgentMetadata(AgentConfigLogsDDURL, cfg.GetString("logs_config.logs_dd_url"))
+	SetAgentMetadata(AgentConfigLogsSocks5ProxyAddress, cfg.GetString("logs_config.socks5_proxy_address"))
+	SetAgentMetadata(AgentConfigNoProxy, cfg.GetStringSlice("proxy.no_proxy"))
+	SetAgentMetadata(AgentConfigProcessDDURL, cfg.GetString("process_config.process_dd_url"))
+	SetAgentMetadata(AgentConfigProxyHTTP, cfg.GetString("proxy.http"))
+	SetAgentMetadata(AgentConfigProxyHTTPS, cfg.GetString("proxy.https"))
```

Comment:
Those might contain secret (ie: http://user:password@proxy). We have a scrubber in `pkg/util/log/strip.go` that scrub everything when creating a flare.

Also I think for this part we might want a test to make sure we will never send password from the config (in case the scrubber change or something else).

Reply:
Ah, good catch -- thanks!